.gitignore

```diff
@@ -72,3 +72,4 @@ CMakeSettings.json
 
 # VisualStudio project files
 /.vs/
+.vscode/
```

Comment:
This should go in your local `.git/info/exclude`, not the shared project `.gitinfo`. Not everyone uses VSCode.